assets/js/icons/stories/index.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * External dependencies
+ */
+import { omitBy, omit, map } from 'lodash';
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Icon from '../icon';
+import * as icons from '../index';
+
+const availableIcons = omit( icons, 'Icon' );
+
+export default {
+	title: 'WooCommerce Blocks/@woocommerce/icons',
+	component: Icon,
+};
+
+const LibraryExample = () => {
+	const [ filter, setFilter ] = useState( '' );
+	const filteredIcons = omitBy( availableIcons, ( _icon, name ) => {
+		return name.indexOf( filter ) === -1;
+	} );
+	return (
+		<div style={ { padding: '20px' } }>
+			<label htmlFor="filter-icons" style={ { paddingRight: '30px' } }>
+				Filter Icons
+			</label>
+			<input
+				// eslint-disable-next-line no-restricted-syntax
+				id="filter-icons"
+				type="search"
+				value={ filter }
+				placeholder="Icon name"
+				onChange={ ( event ) => setFilter( event.target.value ) }
+			/>
+			<div
+				style={ {
+					display: 'flex',
```

Comment:
What's the benefit of having inline styles instead of a `style.scss` file? The `QuantitySelector` component has a [`stories/style.scss` file](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/main/assets/js/base/components/quantity-selector/stories/style.scss), so it might be good to decide what's the best approach and be consistent with it.

Reply:
This is just a story and not a component we use so I'm just using inline styles for the purpose of the story :)

Reply:
>  The QuantitySelector component has a stories/style.scss file, so it might be good to decide what's the best approach and be consistent with it.

In that case the _css class_ is being modified for the selector on the component.  Personally I think we should avoid styles on actual components as much as possible in stories (favoring what the _default_ look for the component is). That will help us catch potential issues with the default not being usable.